### PR TITLE
Add pagination to docs

### PR DESCRIPTION
**Pull request by sebwoj**
Signed-off-by: Sebastian Wojciechowski <s.wojciechowski89@gmail.com>

### CODE REVIEW

**Review by Zlopez — DISMISSED** [review]
LGTM

**Comment by bowlofeggs** [comment]
I'm going to rebase and force push this.

File: docs/user/man_pages/bodhi.rst [hunk 1]
```diff
@@ -133,6 +133,13 @@ The ``overrides`` command allows users to manage build overrides.
 
         Filter for overrides by a list of usernames, given as a comma-separated list.
 
+      ``--rows <integer>``
```
Comment (bowlofeggs) [inline docs/user/man_pages/bodhi.rst]:
```suggestion
    ``--rows <integer>``
```

File: docs/user/man_pages/bodhi.rst [hunk 2]
```diff
@@ -133,6 +133,13 @@ The ``overrides`` command allows users to manage build overrides.
...
+
+        Limits number of results shown per page.
+
+      ``--page <integer>``
```
Comment (bowlofeggs) [inline docs/user/man_pages/bodhi.rst]:
```suggestion
    ``--page <integer>``
```

File: docs/user/man_pages/bodhi.rst [hunk 3]
```diff
@@ -418,6 +425,14 @@ The ``updates`` command allows users to interact with bodhi updates.
 
         Filter for updates by a list of usernames, given as a comma-separated list.
 
+      ``--rows <integer>``
```
Comment (bowlofeggs) [inline docs/user/man_pages/bodhi.rst]:
```suggestion
    ``--rows <integer>``
```

File: docs/user/man_pages/bodhi.rst [hunk 4]
```diff
@@ -418,6 +425,14 @@ The ``updates`` command allows users to interact with bodhi updates.
...
+
+        Limits number of results shown per page.
+
+      ``--page <integer>``
```
Comment (bowlofeggs) [inline docs/user/man_pages/bodhi.rst]:
```suggestion
    ``--page <integer>``
```

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
The new options render inconsistently with the existing options due to indentation. It's an easy fix - I'll fix and push to your branch. Thanks for the patch!

**Comment by bowlofeggs** [comment]
I'm going to apply this patch to your branch:

```
diff --git a/docs/user/man_pages/bodhi.rst b/docs/user/man_pages/bodhi.rst
index 1430170f..c8b694c7 100644
--- a/docs/user/man_pages/bodhi.rst
+++ b/docs/user/man_pages/bodhi.rst
@@ -133,11 +133,11 @@ The ``overrides`` command allows users to manage build overrides.

         Filter for overrides by a list of usernames, given as a comma-separated list.

-      ``--rows <integer>``
+    ``--rows <integer>``

         Limits number of results shown per page.

-      ``--page <integer>``
+    ``--page <integer>``

         Go to page number.

@@ -425,11 +425,11 @@ The ``updates`` command allows users to interact with bodhi updates.

         Filter for updates by a list of usernames, given as a comma-separated list.

-      ``--rows <integer>``
+    ``--rows <integer>``

         Limits number of results shown per page.

-      ``--page <integer>``
+    ``--page <integer>``

         Go to page number.
```

**Review by bowlofeggs — APPROVED** [review]

**Comment by bowlofeggs** [comment]
This feature is planned for inclusion in the upcoming 3.11.0 release: https://github.com/fedora-infra/bodhi/pull/2709

**Comment by bowlofeggs** [comment]
Bodhi 3.11.0b5 has been deployed to staging:

https://bodhi.stg.fedoraproject.org

It is also available via Copr:

https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release

**Pull request merged**